src/hu/elte/txtuml/stdlib/util/UUIDImplementation.java

```diff
@@ -0,0 +1,112 @@
+package hu.elte.txtuml.stdlib.util;
+
+/**
+ * Implementation class for UUID interface for txtUML models.
+ * 
+ * @see UUID
+ * @author Salhi Fadhel (fadhelsalhif@gmail.com)
+ *
+ */
+
+public class UUIDImplementation implements UUID {
+
+	/**
+	 * Store UUID object and then do all other functions with it
+	 */
+	private java.util.UUID var;
+
+	/**
+	 * Instance method that returns val,
+	 * 
+	 * @param val
+	 * @return
+	 */
+	protected java.util.UUID getOrigUUID(Object val) {
```

Comment:
What is the use of this function? Shouldn't this just return the `var` field of `this`?